e2e/fixtures/loginAsAdmin.ts

```diff
@@ -0,0 +1,26 @@
+import { Page, PlaywrightTestArgs, TestFixture } from '@playwright/test';
+import { LoginPage } from '../pages';
+
+/**
+ * A fixture which automatically logs in as the configured admin.
+ *
+ * You can use the fixture as a substitute for the default playwright `page`:
+ * ```ts
+ * test('your test', async ({ loginAsAdmin: page }) => {
+ *   // `page` starts at the OpenMRS SPA's home page.
+ *   // The configured admin user is already logged in.
+ *   await page.foo(); // Your code here.
+ * });
+ * ```
+ */
+export const loginAsAdmin: TestFixture<Page, PlaywrightTestArgs> = async ({ page }, use) => {
```

Comment:
The login functionality has been pre-configured to reuse the signed-in state, eliminating the need for a fixture. 

For reference, you can check the global-setup.ts file located in the e2e/core directory in this repository: https://github.com/openmrs/openmrs-esm-patient-management/blob/main/e2e/core/global-setup.ts

For more information on reusing signed-in state, please refer to this documentation: https://playwright.dev/docs/auth#reuse-signed-in-state"

Reply:
I hard corded this to manually load the login page because the patient search test hanged at the login page.   

> For more information on reusing signed-in state, please refer to this documentation: https://playwright.dev/docs/auth#reuse-signed-in-state"

thank you for this. I'll share feedback shortly